Cargo.toml

```diff
@@ -21,6 +21,7 @@ version = "0"
 [features]
 rt = ["stm32l4x6/rt"]
 STM32L476VG = []
+STM32L496AG = []
 
 [package.metadata.docs.rs]
-features = ["STM32L476VG", "rt"]
+features = ["STM32L476VG", "STM32L496AG", "rt"]
```

Comment:
As per our last talk you can leave figuring out docs to me so remove one of the board crates from docs.rs metadata